### PR TITLE
Include entity-less "scans.json" into an example of inheritance principle

### PR DESCRIPTION
**Pull request by yarikoptic**
- Clarifies #789

### CODE REVIEW

**Comment by effigies** [comment]
I guess I misunderstood the previous discussion. I thought we did not need to single out `scans.json`?

This seems fine, though.

**Comment by yarikoptic** [comment]
it is just an example to make it clearer - nothing is singled out IMHO.

**Comment by yarikoptic** [comment]
> This seems fine, though.

please bless it then with an approval ;-)

**Review by effigies — APPROVED** [review]

**Review by Remi-Gau — APPROVED** [review]

File: src/common-principles.md [hunk 1]
```diff
@@ -773,6 +775,8 @@ the value for field "`RepetitionTime`" is therefore overridden to the value `3.0
 The value for field "`EchoTime`" remains applicable to that image, and is not unset by its
 absence in the metadata file at the lower level (rule 5.b; corollary 3).
 
+A single `scans.json`, without any entity in the filename at the top level, is applicable to describe columns of the `sub-01_scans.tsv` and any other `_scans.tsv` potentially present in the dataset for other subjects.
+
```
Comment (Remi-Gau) [inline src/common-principles.md:779]:
```suggestion
A single `scans.json`, without any entity in the filename at the top level,
is applicable to describe columns of the `sub-01_scans.tsv`
and any other `_scans.tsv` potentially present in the dataset for other subjects.

```

**Comment by Remi-Gau** [comment]
added some semantic linebreak

**Comment by effigies** [comment]
Added the `exclude-from-changelog` label because this doesn't affect the semantic meaning of the spec. For the sake of readable changelogs, I'm trying to be pretty stingy about what changes make it there.

Please let me know if you disagree.

**Pull request merged**